lib/elastic_apm/instrumenter.rb

```diff
@@ -256,10 +256,11 @@ def update_transaction_metrics(transaction)
         tags: tags, reset_on_collect: true
       ).inc!
 
-      tags.merge!('span.type': 'app')
```

Comment:
This was merging into all the metrics above too as they all had a ref to the same hash.